pdf/config.json5

```diff
@@ -1124,45 +1142,7 @@ with open(sys.argv[1], 'w') as html: \n\
                 ],
             },
 
-            pdfium: {
```

Comment:
Since Pdfium didn't work locally, it makes sense that you deleted this, but a merge back to master would need to retain this block. We can talk about it -- maybe the best way forward would be to have a local FAW file which can specify that certain blocks should be skipped?